metrics/sinks/gcm/gcm.go

```diff
@@ -28,15 +27,12 @@ import (
 	"github.com/golang/glog"
 	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/google"
-	gcm "google.golang.org/api/cloudmonitoring/v2beta2"
-	gcm3 "google.golang.org/api/monitoring/v3"
+	v3 "google.golang.org/api/monitoring/v3"
```

Comment:
Please leave gcm name here. This was named explicitly to avoid confusion with anything that can be named v3